FiveCalls/FiveCalls/AppDelegate.swift

```diff
@@ -30,7 +30,12 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         if !UserDefaults.standard.bool(forKey: UserDefaultsKeys.hasShownWelcomeScreen.rawValue) {
             showWelcome()
         }
-        
+
+        var types = UIUserNotificationType()
+        types.insert(.alert)
+        types.insert(.badge)
+        UIApplication.shared.registerUserNotificationSettings(UIUserNotificationSettings(types: types, categories: nil))
```

Comment:
actually, @subdigital isn't [UIUserNotification](https://developer.apple.com/reference/uikit/uiusernotificationaction) the framework that was *deprecated* in iOS 10? I was trying to avoid the new [UNNotification](https://developer.apple.com/reference/usernotifications/unnotification) framework.